frontend/src/utils/capitalizeFirstLetter.js

```diff
@@ -0,0 +1,10 @@
+const capitalizeFirstLetter = (string) => {
+  var splitString = string.toLowerCase().split(" ");
+  for (var i = 0; i < splitString.length; i++) {
+    splitString[i] =
+      splitString[i].charAt(0).toUpperCase() + splitString[i].substring(1);
+  }
+  return splitString.join(" ");
+};
+
+export default capitalizeFirstLetter;
```

Comment:
much easy if you can do it with css
`text-transform:capitalize;`

or this one line
```
function capitalizeFirstLetter(string) {
  return string.charAt(0).toUpperCase() + string.slice(1);
}
```